evaluate.py

```diff
@@ -52,7 +52,7 @@ def evaluate(test_csvs, create_model, try_loading):
                                                  output_classes=tfv1.data.get_output_classes(test_sets[0]))
     test_init_ops = [iterator.make_initializer(test_set) for test_set in test_sets]
 
-    (batch_x, batch_x_len), batch_y = iterator.get_next()
+    batch_wav_filename, (batch_x, batch_x_len), batch_y = iterator.get_next()
```

Comment:
The `get_next()` call in `DeepSpeech.py` needs to be changed as well, no? If only to throw away the value.

Reply:
Yeah looks like it failed the training test because of this.

Reply:
Ah yeah - I never did a training with this... will fix it.